src/python/WMCore/Database/ExecuteDAO.py

```diff
@@ -259,13 +270,30 @@ def main():
     """
     An Utility to construct a DAO Factory and execute the DAO requested.
     """
+    daoObject = ExecuteDAO(logger=logger, configFile=configFile,
+                           package=args.package, daoModule=args.module)
+    daoObject(*args.sqlArgs, dryRun=args.dryRun, daoHelp=True, **args.sqlKwArgs)
+
+
+if __name__ == '__main__':
     args = parseArgs()
 
     if args.debug:
         logger = loggerSetup(logging.DEBUG)
     else:
         logger = loggerSetup()
 
+    # Create an instance of the *.pkl file provided with the dao call, ifa any.
+    if args.pklFile:
+        pklFilePath = os.path.normpath(args.pklFile)
+        if not os.path.exists(pklFilePath):
+            logger.error(f"Cannot find the pkl file: {pklFilePath}. Exit!")
+            exit(1)
```

Comment:
I think you have to provide the library name as well, hence `sys.exit(1)`, no?

Reply:
done